domains/tekaje.json

```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "ZarXD",
+           "email": "banghads09@gmail.com",
+           "discord": "835526814088298526"
+        },
+    
+        "record": {
+            "CNAME": "zarxd.github.io/tkj-one-gallery"
```

Comment:
invalid cname (anly include a hostname or domain and dont include path like /tkj-one-gallery)